### DISCUSSION TITLE
Rename without a Git repository

### DISCUSSION THREAD

**Discussion by kotx** [ModernRonin/ProjectRenamer#45] [Ideas]
Could be a flag like `--no-git`.

**Comment by ModernRonin** [comment]
can you elaborate what you would want this flag to do?
thanks,
MR

**Comment by ModernRonin** [comment]
ah, never mind, I didn't read the title of the issue.
okay, I'll try and add that, but it might take a while until I got time.

**Comment by jamesbraza** [comment]
Another use case for this: monorepos (containing a Visual Studio project that doesn't match the repo name)
